flixel/graphics/tile/FlxDrawTilesItem.hx

```diff
@@ -73,6 +80,11 @@ class FlxDrawTilesItem extends FlxDrawBaseItem<FlxDrawTilesItem>
 				tempFlags |= Tilesheet.TILE_RGB;
 			}
 			
+			if (hasColorOffsets)
+			{
+				tempFlags |= Tilesheet.TILE_TRANS_COLOR;
```

Comment:
I can guard it using ```!openfl_legacy``` and check for openfl version 3.5.4 (the next tag), but Joshua hasn't applied that yet. I just pinged him about it, tho.